packages/components/src/tabs/docs/loader.css

```diff
@@ -17,6 +17,15 @@
     border-radius: 50px;
     border: 2px solid var(--o-ui-b-alias-low-break);
     border-top-color: var(--o-ui-white);
-    animation: o-ui-button-spinner 1s linear infinite;
+    animation: o-ui-sb-spinner 1s linear infinite;
     transform: translate(-50%, -50%) rotate(0deg);
 }
+
+@keyframes o-ui-sb-spinner {
```

Comment:
is this spinner specific to a tab docs example or any other exemple in the docs could use it?

Reply:
Any other examples could use it

Reply:
I guest this could be moved then

Reply:
yeah should go in `docs.css`